modules/nw-operator-cr.adoc

```diff
@@ -284,6 +284,11 @@ endif::operator[]
  While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
 ====
 
+
+|`gatewayConfig.ipForwarding`
```

Comment:
shouldn't this be just "ipForwarding" and down in the section where gatewayConfig is (line 338)? Like where routingViaHost is?

Reply:
@trozet Moved. Thanks a lot!